libmscore/rendermidi.cpp

```diff
@@ -1470,7 +1470,7 @@ void Score::createPlayEvents(Chord* chord)
       int unit = st.swingUnit;
       int ratio = st.swingRatio;
       // Check if swing needs to be applied
-      if (unit) {
+      if (unit && !chord->tuplet()) {
```

Comment:
I would have put the test in the swingAdjustParams function, just in case we would use it somewhere else.